CONTRIBUTING.rst

```diff
@@ -145,75 +145,75 @@ Examples:
 Additional Info on Github
 =========================
 
-The following github documentation may be of use:
+The following GitHub documentation may be of use:
 
 - See `Using Pull Requests
-  <https://help.github.com/articles/using-pull-requests>`_
-  for more information about Pull Requests.
+ <https://help.github.com/articles/using-pull-requests>`_
+ for more information about Pull Requests.
 
 - See `Fork A Repo <http://help.github.com/forking/>`_ for an
-  introduction to forking a repository.
+ introduction to forking a repository.
 
 - See `Creating branches
-  <https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/>`_
-  for an introduction on branching within GitHub.
+ <https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/>`_
+ for an introduction on branching within GitHub.
 
 
 Developing and committing
 *************************
 
 .. contents::
-   :local:
+  :local:
 
 Guidelines for developing
 =========================
 
 - Make sure the PR changes are not in conflict with the documentation,
-  either documentation files (`/README.md`, `/documentation/`), program help,
-  or ADS Wiki.
+ either documentation files (`/README.md`, `/documentation/`), program help,
+ or ADS Wiki.
 
-  If conflict, address them.
+ If conflict, address them.
 
 - Please add tests, especially with new code:
 
-  #TODO: UPDATE PARAGRAPH FOR ADS As of now, we have integration tests (that run in `sct_testing`),
-  and unit tests (in `/unit_testing/`).
+ #TODO: UPDATE PARAGRAPH FOR ADS As of now, we have integration tests (that run in `sct_testing`),
```

Comment:
For ADS, we can say:
« 
As of now, we have integration tests and unit tests in the ``./test/`` folder.